CondorMonitoring/JobCounter.py

```diff
@@ -589,72 +604,99 @@ def main():
+                    else:
+                        print 'How did you got here?! Are you developing something?'
+                        raise
+                    try:  # it can be an ExprTree
+                        cpus = get_int(job['RequestCpus'])
+                    except:
+                        # Catch any except in case something in the future would change in HTCondor or how ExprTree is evaluated.
+                        # It is not correct to assume it is 1 cpu. Skip this job.
+                        print 'Failed to extract RequestCpus from this job %s' % job
+                        continue
+                    siteToExtract = str(job['DESIRED_Sites']).replace(' ', '').split(",")
+                    while '' in siteToExtract:
+                        siteToExtract.remove('')
+                    if not siteToExtract:
+                        # There are some cases in CRAB, which it makes to have zombie jobs without any DESIRED_Sites.
+                        # See here: https://github.com/dmwm/CRABServer/issues/4933
+                        # Skip it as it will not be able to run anywhere..
+                        print 'No Sites... %s' % job
                         continue
 
-                if status == 2:  # Running
-                    increaseRunning(siteToExtract[0], schedd_name, jType, cpus)
-                    increaseRunningWorkflow(workflow, siteToExtract[0], 1)
-                elif status == 1:  # Pending
-                    pendingCache.append([schedd_name, jType, cpus, siteToExtract])
-                    increasePendingWorkflow(workflow, siteToExtract, 1)
-                else:  # Ignore jobs in another state
-                    continue
+                    if schedd_name in relvalAgents:  # If RelVal job
+                        jType = 'RelVal'
+                    elif task == 'Reco':  # If PromptReco job (Otherwise type is Processing)
+                        jType = 'Reco'
+                    elif jType not in jobTypes:  # If job type is not standard
+                        jType = jobType(jobId, schedd_name, task)
+
+                    siteRunning = job.get(jobKeys[schedd_type]['sitename'], '')
+                    if siteName(siteRunning) and status == 2:  # If job is currently running
+                        siteToExtract = [siteRunning]
+
+                    # Ignore jobs to the T1s from the Tier-0 pool
+                    # Avoid double accounting with the global pool
+                    if collector_name == tier0_pool[0]:
+                        if not 'T2_CH_CERN_T0' in siteToExtract:
+                            continue
+
+                    # Ignore jobs to T2_CH_CERN_T0 from the global pool
+                    # Avoid double accounting with the Tier-0 pool
+                    if collector_name == global_pool[0]:
+                        if 'T2_CH_CERN_T0' in siteToExtract:
+                            continue
+
+                    if status == 2 and siteRunning:  # Running
```

Comment:
this job status check is already done above, we should converge them into a single one.